docs/hub/security-tokens.md

```diff
@@ -0,0 +1,59 @@
+---
+title: User Access Tokens
+---
+
+<h1>User access tokens</h1>
+
+
+## What are User Access Tokens?
```

Comment:
I feel some of the sections in this page could be subsections under security-tokens

Reply:
Putting this in the backlog so that I can try splitting it up in the new UI to see how it feels